client/dashboard/sites/overview/site-card.tsx

```diff
@@ -8,23 +9,24 @@ import {
 } from '@wordpress/components';
 import { dateI18n } from '@wordpress/date';
 import { __, sprintf } from '@wordpress/i18n';
+import { sitePHPVersionQuery } from '../../app/queries';
 import { getSiteStatusLabel } from '../../utils/site-status';
 import SitePreview from '../site-preview';
 import type { Site, Plan } from '../../data/types';
 
+function TextBlur( { text }: { text: string } ) {
+	return <span className="text-blur" data-text={ text } />;
```

Comment:
This class should have been `dashboard-overview-site-card__text-blur` or something like that.

Reply:
The current class is way too generic.

Reply:
I never understand how these are named. Anyway, it's moved to components now. I guess it should be `dashboard-text-blur`, looking at those other components?

Reply:
>  I guess it should be dashboard-text-blur, looking at those other components?

yes I think so